torch/csrc/jit/tensorexpr/tensorexpr_init.cpp

```diff
@@ -110,22 +110,33 @@ void initTensorExprBindings(PyObject* module) {
   py::class_<BufHandle, ExprHandle>( // NOLINT
       te,
       "BufHandle")
-      .def(py::init<
-           const std::string&,
-           const std::vector<ExprHandle>&,
-           Dtype>());
+      .def(
+          py::init<const std::string&, const std::vector<ExprHandle>&, Dtype>())
+      .def(
+          "load",
+          [](BufHandle& self,
+             const std::vector<ExprHandle>& v) {
+            return Load::make(self, v);
+          });
 
   py::class_<Placeholder>(te, "Placeholder")
       .def(py::init<
            const std::string&,
            const Dtype&,
            std::vector<ExprHandle>&>())
-      .def("load", [](Placeholder& self, const std::vector<ExprHandle>& v) {
-        return self.load(v);
-      });
+      .def(
+          "load",
+          [](Placeholder& self, const std::vector<ExprHandle>& v) {
+            return self.load(v);
+          })
+      .def(
+          "buf",
+          [](Placeholder& self) { return BufHandle(self.data()); },
+          py::return_value_policy::reference);
```

Comment:
Do we need this `return_value_policy`? I'd think it wouldn't be necessary since we're returning a handle.